### PR TITLE
Update images to use Golang 1.12.5 and remove kind

### PR DESCRIPTION
**Pull request by ericvn**
- gcr.io/istio-testing/istio-builder:{date}

Also updated Bazel to 0.26.0

Once this is merged and the images built (also need date tag of version pushed), another PR can be merged to specify the new images in the yams files.

### CODE REVIEW

File: docker/istio_builders/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PROJECT = istio-testing
-VERSION = 0.5.11
+VERSION = 0.5.12
```
Comment (icygalz) [inline docker/istio_builders/Makefile]: revert this changes as this directory is not used anymore.

**Review by icygalz — COMMENTED** [review]
Do we want to update bazel for prowbazel in this PR also? They already have bazel 0.26.0 (https://github.com/bazelbuild/bazel/releases). If yes, you can change it here:
https://github.com/istio/test-infra/blob/master/scripts/tools/linux-install-bazel#L11

If not, you can disregard this comment :).

File: docker/istio_builders/CHANGELOG.md [hunk 2]
```diff
@@ -9,3 +9,4 @@
 * 0.5.9: update docker to use overlay2 fs for performance and update bazel to 0.16.1
 * 0.5.10: update entrpypoint
 * 0.5.11: update gcc/g++ to 7.0
+* 0.5.12: update golang to 1.12.5
```
Comment (icygalz) [inline docker/istio_builders/CHANGELOG.md]: revert this one also.

File: docker/istio_builders/Makefile (hunk 1 above)
Comment (ericvn) [reply docker/istio_builders/Makefile]: done

File: docker/istio_builders/CHANGELOG.md (hunk 2 above)
Comment (ericvn) [reply docker/istio_builders/CHANGELOG.md]: done

**Comment by ericvn** [comment]
I also did the Bazel update.

File: docker/prowbazel/README.md [hunk 3]
```diff
@@ -49,3 +49,4 @@ Our dependency on this script is because it appropariately writes test job resul
 * 0.5.2: Update gcc and g++ to 7.0
 * 0.5.3: Update cmake to 3.8.0
 * 0.5.4: Update ninja to 1.9.0
+* 0.5.5: Update golang to 1.12.5
```
Comment (icygalz) [inline docker/prowbazel/README.md]: can you also add 'update bazel to 0.26.0' here?

Comment (ericvn) [reply docker/prowbazel/README.md]: done

**Review by icygalz — APPROVED** [review]

**Comment by ericvn** [comment]
Part of https://github.com/istio/istio/issues/14420

**Comment by icygalz** [comment]
I forgot that we actually need to update 1 more file for upgrading bazel to 0.26.0 which i have done in this PR:
https://github.com/istio/test-infra/pull/1296

I have pushed these images containing this PR changes and https://github.com/istio/test-infra/pull/1296:
gcr.io/istio-testing/prowbazel:0.5.5
gcr.io/istio-testing/istio-builder:v20190531-aba3be6
gcr.io/istio-testing/test-infra-builder:v20190531-6fbeace

You can now make the PR for using these images.
